packages/dicomImageLoader/src/types/ImageFrame.ts

```diff
@@ -0,0 +1,32 @@
+interface ImageFrame {
+  samplesPerPixel: number;
+  photometricInterpretation: string;
+  planarConfiguration: number;
+  rows: number;
+  columns: number;
+  bitsAllocated: number;
+  bitsStored: number;
+  pixelRepresentation: number;
+  smallestPixelValue: number;
+  largestPixelValue: number;
+  redPaletteColorLookupTableDescriptor: number[];
+  greenPaletteColorLookupTableDescriptor: number[];
+  bluePaletteColorLookupTableDescriptor: number[];
+  redPaletteColorLookupTableData: number[];
```

Comment:
These can also be bulkdata URIs which need to be loaded, although it looks like that has gotten broken at some point.

Reply:
So you are saying this can be `number[] | string`?